server/scan/device_handling.py

```
@@ -1,764 +1,816 @@
+        # Fetch CurrentScan data
+        query = """
+            SELECT cur_MAC, cur_Name, cur_Vendor, cur_ScanMethod, cur_IP, cur_SyncHubNodeName,
+                   cur_NetworkNodeMAC, cur_PORT, cur_NetworkSite, cur_SSID, cur_Type
+            FROM CurrentScan
+        """
+        db.sql.execute(query)
+        current_scan_data = db.sql.fetchall()
+        
+        for row in current_scan_data:
+            cur_MAC, cur_Name, cur_Vendor, cur_ScanMethod, cur_IP, cur_SyncHubNodeName, \
+            cur_NetworkNodeMAC, cur_PORT, cur_NetworkSite, cur_SSID, cur_Type = row
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)